src/core/engine/processors/mongo_to_postgresql_order_status_history.py

```diff
@@ -71,6 +73,23 @@ def load_data(self, transformed_data):
             mongo_uuids.append(
                 ObjectId(postgres_order_status_history.mongo_order_status_history_uuid)
             )
+
+            if postgres_order_status_history.from_status != OrderStatus.NEW_ORDER.name:
+                last_order_status_history = self.order_status_history_controller.get_last_status_history_by_order_id(
```

Comment:
You should get all last status order histories by an array of order ids

Reply:
done.